packages/perseus/src/widgets/unit.jsx

```diff
@@ -179,16 +178,7 @@ export class OldUnitInput extends React.Component<Props> {
 
     // TODO(joel) think about showing the error buddy
     render(): React.Node {
-        const input = this.props.apiOptions.staticRender ? (
-            <MathOutput
-                onChange={this.handleChange}
-                ref="input" // eslint-disable-line react/no-string-refs
-                className={ApiClassNames.INTERACTIVE}
-                value={this.props.value}
-                onFocus={this.handleFocus}
-                onBlur={this.handleBlur}
-            />
-        ) : (
+        const input = (
```

Comment:
🤔  I think this is now a cycle.  The `input` on line 182 seems to refer to itself (line 181).  Can you maybe do a super-simple unit test or story to see if this component actually renders?

Reply:
Would you call it a _Unit_ test? Because it's the Unit widget? 😅

https://github.com/Khan/perseus/pull/423

It was working in Storybook as-is, but I see what you mean. Removed that cycle, added a unit test, and cleaned up the component while I was there.

Reply:
> Would you call it a Unit test? 
Well played! 😂